client/src/constants.tsx

```diff
@@ -8,7 +8,7 @@ export const LOCATIONS_UNIT = 'Locations unit';
 export const LOCATIONS_UNIT_GROUP = 'Locations unit group';
 export const LOCATIONS_UNIT_GROUP_SET = 'Locations unit group set';
 export const PRODUCT_CATALOGUE = 'Product Catalogue';
-export const PLANS = 'Plans';
+export const PLANS = 'Missions';
```

Comment:
I feel weird about this.  Why not change to 

```ts
export const MISSIONS = 'Missions';
```

Reply:
I thought of consistently naming plans knowing this will be translated in the future on a project basis